src/include/kernel/ext_manifest.h

```diff
@@ -104,4 +105,11 @@ struct ext_man_probe_support {
 	struct sof_ipc_probe_support probe;
 } __packed;
 
+/* IPC MSG max size */
+struct ext_man_ipc_msg_size {
```

Comment:
ok, I guess this aligns with EXT_MAN_ELEM_IPC_SIZE. Can we add each enum name to the comments above the struct where its is used.

Reply:
Small update for names so its clear now: ext_man_ipc_size for EXT_MAN_ELEM_IPC_SIZE.
Also added inbox/outbox sizes